tasties_app/tests/test_recipe.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+from tasties_app.models import Recipe, Category
+from django.utils import timezone
+from django.contrib.auth.models import User
+
+
+@pytest.fixture
+def recipe():
```

Comment:
I think this too long for fixture,  split your fixture, for  example do the user creation as fixture and then
put it as an argument in this fixture

Reply:
I tried doing what you suggest but it doesn't work. The information is not saved and it gives me errors. If you could help me with this I would be very grateful.